presto-orc/src/main/java/com/facebook/presto/orc/reader/LongDictionaryBatchStreamReader.java

```diff
@@ -28,14 +28,14 @@
 import com.facebook.presto.orc.stream.InputStreamSource;
 import com.facebook.presto.orc.stream.InputStreamSources;
 import com.facebook.presto.orc.stream.LongInputStream;
+import javafx.util.Pair;
```

Comment:
Please don't use this package, it belong to UI. Java folks typically stay away from any Pair/Tuple classes.